### ISSUE TITLE
Can't use cards from other seed.

### ISSUE THREAD

**Issue by zoltancservak** [OndrejNepozitek/PlateUpMods#4]
Is there maybe a way to use a given seed's cards, like variety and sides as a part of the special kitchen layout? I want a big kitchen with all sides+variety, i have a side focused seed that gives variety.

**Comment by OndrejNepozitek** [comment]
Hello! Have you tried the `FixedSeed` option in the Configuration Manager? It can be seen in this screenshot:

![image](https://user-images.githubusercontent.com/2265359/195846339-0ff40482-3245-47b0-bfdb-df196f90deee.png)

You can uncheck the `UseRandomSeed` checkbox and input your desired seed into the `FixedSeed` field. That should make it possible to play with any seed you want. 

But I have not tested very thoroughly so any feedback is appreciated!

**Comment by zoltancservak** [comment]
Yes i have, it looked like the blueprints somewhat aligned, but not the cards. I will try it again later.

**Comment by zoltancservak** [comment]
Yeah, just tested and the variety cards aren't the same, not sure about the customer cards and so on, but the ones you choose at day 15 aren't the same.

**Comment by OndrejNepozitek** [comment]
Can you please describe the exact setup that you tried? I could try to reproduce it myself.

**Comment by OndrejNepozitek** [comment]
So, I did some testing. Is it possible that you entered the seed in uppercase letters? E.g. `blargh` vs `BLARGH`. It looks like if you are using the normal vanilla way of entering seeds, the seed is probably automatically converted to lowercase letters. But if you use Kitchen designer and enter the seed into the text box field, the seed is not converted to lowercase. Therefore, you would get different results. It is a bit misleading that the game displays the seeds in uppercase letters but the are internally lowercase.

I tried one fixed seed with a vanilla layout and then used the same seed with Kitchen designer and I got the exact same cars and franchise options.
